doc/building_picoquic.md

```diff
@@ -0,0 +1,102 @@
+# Building PicoQUIC
+
+See the 'Building Picoquic' section [README](../README.md) for building picoquic for the first time and installing the dependencies.
+
+## Dependencies
+
+Direct dependencies
+- [Picotls implementation of TLS 1.3](https://github.com/h2o/picotls)
+
+Fetching and installing this package can be done through an option in the CMake.
+
+```shell
+ cmake -DPICOQUIC_FETCH_PTLS=Y .
+```
+
+This can be set in the cmake file as well by setting the `PICOQUIC_FETCH_PTLS` cmake property to `ON`.
+
```

Comment:
The command on line 13 sets `PICOQUIC_FETCH_PTLS` to `Y`. Line 16 suggests setting to `ON`. The GiHub action code uses:
```shell
cmake -S . -B build -DPICOQUIC_FETCH_PTLS:BOOL=ON
```
My suggestion would be to say something like:

Fetching and installing this package can be done by launching CMake with a fetch option:

```shell
 cmake -DPICOQUIC_FETCH_PTLS=Y .
```

And not adding the extra line 16, which seems to encourage people to edit the CMake file manually.